.devcontainer/dapr-setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Install dapr CLI
+wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash
+
+# Initialize Dapr
+sudo dapr init
+
+# Install Go
+wget -q -O - https://raw.githubusercontent.com/canha/golang-tools-install-script/master/goinstall.sh | bash
```

Comment:
Can we use this extension instead: `ms-vscode.go`

Reply:
@pruthvidhodda 
Because Go tools are not installed by default on this container. Go is needed for samples like distributed calculator

Reply:
Oh I missed that one of the samples need Go - sorry.